app/services/cert/certs.rb

```diff
@@ -4,8 +4,12 @@ def self.ueber_cert(organization)
       organization.debug_cert
     end
 
+    def self.ca_cert_filename
+      Setting[:ssl_ca_file]
```

Comment:
but… is that the right file?
it's `/etc/foreman/proxy_ca.pem`, which is the one we use to talk to proxies.
but here, were talking to Pulpcore via Apache, and Apache will be using the server-ca?

@ehelms halp

Reply:
I think so, but I still prefer https://github.com/Katello/katello/pull/8832 which defaults to that file while still allowing to override it if needed. It also works it into the ping controller.